Jenkinsfile

```diff
@@ -0,0 +1,101 @@
+#!groovy
+
+timestamps {
+  ansiColor('xterm') {
+    node('node-small') {
+      stage('SETUP') {
+        deleteDir()
+        dir('scm'){
+          checkout scm
+        }
+        dir('solrHome') {
```

Comment:
hmm, I wanted this for solrHome Config, since that's all it's being used for. I'd rather name it based off what it's being used for rather than where we have it internally. this will allow for more flexibility in where the customer may store their solrHome Config.